src/main/java/com/relayrides/pushy/apns/util/ApnsPayloadBuilder.java

```diff
@@ -128,6 +134,47 @@ public ApnsPayloadBuilder setLocalizedAlertMessage(final String localizedAlertKe
 	}
 
 	/**
+	 * <p>Sets a short description of the notification purpose. The Apple Watch will display this as part of the notification.
+	 * The string is visible for a short amount of time and should be crafted to be easy to understand.
```

Comment:
Minor, but it looks like you're missing a closing `</p>` tag here.

Also, rather than paraphrasing the docs, I think it makes more sense to quote directly (with attribution!) like we do for `setLaunchImage`. I know I haven't been super consistent on this point and apologize for that, but it seems like a good direction to go for new docs.